src/lightning/pytorch/_graveyard/precision.py

```diff
@@ -0,0 +1,57 @@
+import sys
+
+import lightning.pytorch as pl
+from lightning.pytorch.plugins.precision import (
+    BitsandbytesPrecision,
+    DeepSpeedPrecision,
+    DoublePrecision,
+    FSDPPrecision,
+    HalfPrecision,
+    MixedPrecision,
+    Precision,
+    TransformerEnginePrecision,
+    XLAPrecision,
+)
+
+
+def _patch_sys_modules() -> None:
+    sys.modules["lightning.pytorch.plugins.precision.precision_plugin"] = sys.modules[
+        "lightning.pytorch.plugins.precision.precision"
+    ]
+
+
+def _patch_classes() -> None:
```

Comment:
If you want to go the extra mile, you could deprecate these classes, let CI run, and then remove the deprecation message.

This will make CI fail if the deprecation message is triggered, and show any leftovers